templates/swiftring/bin/swift-ring-tool

```diff
@@ -122,7 +122,7 @@ function push() {
             '${VERSION}'
             "name":"'${CM_NAME}'",
             "namespace":"'${NAMESPACE}'",
-            "finalizers": ["swift-ring/finalizer"],
+            "finalizers": ["openstack.org/swiftring"],
```

Comment:
could use `helper.GetFinalizer()` when render this script template.

Reply:
Good point, indeed! Will change this in a follow up. Thx!